src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -10,11 +10,16 @@
 import numpy as _np
 from .._tf_model import TensorFlowModel
 import turicreate.toolkits._tf_utils as _utils
-import tensorflow.compat.v1 as _tf
+from turicreate._deps.minimal_package import minimal_package_import_check
 
-# This toolkit is compatible with TensorFlow V2 behavior.
-# However, until all toolkits are compatible, we must call `disable_v2_behavior()`.
-_tf.disable_v2_behavior()
+# in conjunction with minimal package
```

Comment:
Nice!  Let's move this function into _tf_utils as it's copy-pasted elsewhere, then there's only one of it.

Reply:
there's must be a reason it exists everywhere before. If we need to clean it, why now?

Reply:
Didn't you add _lazy_import_tensorflow?  It was added in this PR.

Reply:
Yeah, it's a function name to wrap the existing code.